tools/create/scaffold/vars.go

```diff
@@ -0,0 +1,46 @@
+package scaffold
+
+import "errors"
+
+const (
+	ManifestFileFlagName    = "gen-manifest"
+	ManifestFileFlagDefault = "manifest.yaml"
+
+	ModuleConfigFileFlagName    = "module-config"
+	ModuleConfigFileFlagDefault = "scaffold-module-config.yaml"
+
+	DefaultCRFlagName    = "gen-default-cr"
+	DefaultCRFlagDefault = "default-cr.yaml"
+
+	SecurityConfigFlagName    = "gen-security-config"
+	SecurityConfigFlagDefault = "sec-scanners-config.yaml"
+)
+
+// Flags
+var (
+	ModuleName    string
```

Comment:
`flags.go`. Consts can also go with it. Also you can think about putting description to const as well. Also you could reason to group by flag like this:
```
var ManifestFile string
const (
	ManifestFileFlagName    = "gen-manifest"
	ManifestFileFlagDefault = "manifest.yaml"
)

var ModuleConfigFile string
const (
	ModuleConfigFileFlagName    = "module-config"
	ModuleConfigFileFlagDefault = "scaffold-module-config.yaml"
)
```